src/ripple/overlay/impl/OverlayImpl.h

```diff
@@ -364,6 +373,26 @@ class OverlayImpl : public Overlay
     void
     lastLink(std::uint32_t id);
 
+    /** Updates message count for validator/peer. Sends TMSquelch if the number
+     * of messages for N peers reaches threshold T.
```

Comment:
"Sends TMSquelch if the number of messages for N peers reaches threshold T." 
Is the number of messages >= T or N >= T? Or there are two thresholds?